lib/backends/dummy/index.js

```diff
@@ -78,7 +73,8 @@ var queueDefns = [
             'command_execute',
             'server_reboot',
             'server_sysinfo',
-            'server_update_nics'
+            'server_update_nics',
+            'recovery_config'
```

Comment:
Alphabetical - so place before the "server_reboot" line.

Reply:
Good call